CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `torch`, `gpytorch` and `botorch` are lazy-loaded for improved startup time
 - If an exception is encountered during simulation, incomplete results are returned 
   with a warning instead of passing through the uncaught exception
+- Telemetry no longer requires `requests` package
```

Comment:
Wouldn't it make more sense to move this to "Removed" 🤔

Reply:
as thy wish